classes/item.rb

```diff
@@ -0,0 +1,46 @@
+class Item
+  attr_reader :id, :archived
+  attr_accessor :genre, :author, :source, :label, :published_date
```

Comment:
- [ ] genre, author, source(shouldn't be included since you are three in this group), label should be read only in this class that's `attr_reader :genre, :author, :source, :label`